src/build.sh

```diff
@@ -200,8 +217,8 @@ function load_build_config()
 
 function parse_build_options()
 {
-  local long_options='help,info,menu,doc,ccache,cpu-scaling:,warnings::,save-log-to:,llvm'
-  local short_options='h,i,n,d,c:,w::,s:'
+  local long_options='help,info,menu,doc,ccache,cpu-scaling:,warnings::,save-log-to:,llvm,clean'
+  local short_options='h,i,n,d,S:,w::,s:,c'
```

Comment:
Change `c` to `S` should be in a separate commit before this one. You also need to update the man page.

Reply:
Did you rebase this PR against the latest unstable branch?